File: moda-api/src/main/java/com/moda/moda_api/user/presentation/request/SignupRequest.java
package com.moda.moda_api.user.presentation.request;

import lombok.Getter;
import lombok.NoArgsConstructor;

/**
 * 회원가입 요청을 위한 DTO 클래스입니다.
 */
@Getter
@NoArgsConstructor
public class SignupRequest {
    private String email;
    private String password;
}

File: moda/app/src/main/java/com/example/modapjt/data/dto/request/SignUpRequest.kt
package com.example.modapjt.data.dto.request

data class SignUpRequest(
    val email: String,
    val userName: String,
    val password: String,
    val nickname: String
)

data class EmailVerificationRequest(
    val email: String,
    val code: String? = null
)

data class UserNameRequest(
    val userId: String
)
